app/screens/post_options/post_options.tsx

```diff
@@ -63,17 +72,61 @@ const PostOptions = ({
     const shouldRenderFollow = !(sourceScreen !== Screens.CHANNEL || !thread);
     const shouldShowBindings = bindings.length > 0 && !isSystemPost;
 
-    const snapPoints = [
+    const snapPoints = useMemo(() => {
+        const items: BottomSheetProps['snapPoints'] = [1];
+        let count = shouldShowBindings ? 0.5 : 0;
+        if (canReply && sourceScreen !== Screens.THREAD) {
```

Comment:
```suggestion
        if (canReply) {
```
`sourceScreen !== Screens.THREAD` is already accounted for in the index file. 
<img width="1278" alt="image" src="https://user-images.githubusercontent.com/7575921/210711174-c41ff8a9-aae2-4fd5-b737-1e22d7a680c5.png">